test/test_flow.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+/* eslint-disable */
+
+import * as React from 'react';
+import { DragDropContext, Droppable, Draggable } from '../src';
+
+{
+  <>
```

Comment:
Could you please use `React.Fragment` here? I have been using that throughout the code and it would be good to keep it consistant